docs/fieldnorm/requirements_configuration.md

```diff
@@ -0,0 +1,28 @@
+# Requirements and configuration management
+
+The complexity and variety of developments in the field of bio-informatics, comes with a need for clear prerequisites before the start of a project on the one hand and concise regulation of development versioning on the other. Unfortunately, neither is described for application to medical devices by the ISO-15189, unlike the IEC-62304, which does disclose types of requirements that should be documented.
+
+## 1. Requirements in software development
+The requirements of software should be clear before starting on a project and describe the prerequisites that the software should meet for it to be considered as completed. The requirements may relate to the performance of the desired software, the physical characteristics, the computing environment, a need for compatibility, input- and output-structures, interfaces to other systems, communication from the software, security requirements, user interfaces to the software, data definitions, installation and acceptance requirements, requirements related to the methods of operation and maintenance, network-related requirements, user maintenance requirements and regulatory requirements.
```

Comment:
I would make it a list, instead of concatenation with ','.

Reply:
rephrase: software should meet for it to be considered as completed
(I'll try to think of something)

Reply:
I think you can just remove 'for it to be considered as completed'.